addons/web/static/src/views/list/list_confirmation_dialog.xml

```diff
@@ -33,7 +34,7 @@
             </main>
             <t t-set-slot="footer">
                 <button class="btn btn-primary" t-on-click="_confirm" t-ref="autofocus">
-                Confirm
+                Update All
```

Comment:
This "Update All" button wording is a bit confusing. You're not updating _all_ the records, only the ones you have selected. So maybe this is better?
```suggestion
                Update
```